docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -221,7 +221,21 @@ Custom config is not yet supported for RocksDB.
 
 #### `--genesis` (string):
 
-Path to a JSON file containing the genesis data to use. Ignored when running standard networks (Mainnet, Fuji Testnet), or when `--genesis-content` is specified. If not given, uses default genesis data. For an example of a JSON representation of genesis data, see [here](https://github.com/ava-labs/avalanchego/blob/master/genesis/genesis_local.go#L16).
+Path to a JSON file containing the genesis data to use. Ignored when running standard networks (Mainnet, Fuji Testnet), or when `--genesis-content` is specified. If not given, uses default genesis data.
+
+This are the main properties in the JSON file:
+
+- `networkID`: A unique identifier for the blockchain, must be a number in the range [0, 2^32],
+- `allocations`: The list of initial addresses, their initial balances and the unlock schedule for each.
+- `startTime`: The time of the beginning of the blockchain, must be a timestamp in seconds,
```

Comment:
Is it UNIX epoch time? Am I right that user must specify time in the future and when this time comes nodes start processing transactions?

Reply:
What happens if user specified time in the past?

Reply:
Yes it's Unix epoch time, but it can't be in the future. Please see the next comment.

Reply:
But what is the purpose of `startTime`? Just a note or staking delays are counted from it?

Reply:
Staking are counted from it:
``` go
staker[i].startTime = startTime
staker[i].endTime = startTime + initialStakeDuration - i * initialStakeDurationOffset
```